.github/workflows/rust.yml

```diff
@@ -99,6 +99,11 @@ jobs:
           rust-version: stable
       - name: Run tests (excluding doctests)
         run: cargo test --lib --tests --bins --features avro,json,backtrace
+        env:
```

Comment:
Adding more stack as mem as q54 keeps failing.
```
thread 'tpcds_physical_q54' has overflowed its stack
```
This change shouldn't impact stack as the second param is boxed and thus resides in heap, in stack there is only a pointer.